documentation/service_admin/installing-openshift.adoc

```diff
@@ -153,3 +159,55 @@ The deployments required for running {ProductName} are now created.
 ----
 oc get pods -n enmasse
 ----
+
+===== Deploying {ProductName} for evaluation
```

Comment:
I think of it more as deploying for single tenant use. The terms production/evaluation suggest that the latter is somehow less reliable, which I don't think is actually the issue.

Reply:
@grs I wasn't sure what to call it really. At present I don't think we should set the expectation that it is supported in the same way, because we don't have any CI tests for this mode ( so it kind of is less reliable in that sense).